tests/integration/test_ceph.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# pylint: disable=duplicate-code
+"""Integration tests."""
+
+import pytest
+from juju import model, unit
+
+from .helpers import ready_nodes
+
+# This pytest mark configures the test environment to use the Canonical Kubernetes
+# bundle with ceph, for all the test within this module.
+pytestmark = [
+    pytest.mark.bundle_file("test-bundle-ceph.yaml"),
+    pytest.mark.ignore_blocked,
+]
+
+
+@pytest.mark.abort_on_fail
+async def test_nodes_ready(kubernetes_cluster: model.Model):
```

Comment:
This does not seem ceph related but rather like a smoke test. Do we need this here?

Reply:
Agree, not necessary.  I think the second test is more appropriate